pathways/shipments.py

```diff
@@ -74,6 +74,9 @@ class Shipment(collections.UserDict):
     def __getattr__(self, name):
         return self[name]
 
+    def count_infested(self):
+        return np.count_nonzero(self["stems"])
+
```

Comment:
Great, exactly what I hoped for.

Writing `self.stems` would be preferred, because we want to use the attribute access syntax (`obj.attr`) instead of the dictionary key access (`obj["key"]`). Well, at least according to what I put in the documentation of this class.